packages/studio/__mocks__/virtual_yext-studio.ts

```diff
@@ -5,13 +5,15 @@ const mockFilepath = path.join(__dirname, "../tests/__mocks__");
 const mockStudioData: StudioData = {
   pageNameToPageState: {},
   pageNameToErrorPageState: {},
+  layoutNameToLayoutPath: {},
```

Comment:
nit: should this be `layouts` for now?

Reply:
oh! yes, I missed that-- was from an older implementation.